docstring.py

```diff
@@ -10,7 +10,9 @@ def get_docstring_args(fd):
     Extract docstring parameters from function definition
     """
     docstring = ast.get_docstring(fd)
-    args = re.findall(r"(\w+)\s+\:", docstring)
+    params_section = re.findall(r"(?<=Parameter(.*)(?=Return", docstring, re.DOTALL)[0]
```

Comment:
Looks like you are missing two closing parantheses:

```
re.findall(r"(?<=Parameters)(.*)(?=Returns)"
```

Note that `(?<=Parameters)` and `(?=Returns)` are unique for regex as they are considered "look ahead"/"look behind" methods and both need their own parantheses

Reply:
Right! I missed that. Will fix it.